io/las/LasReader.cpp

```diff
@@ -67,6 +67,7 @@ class invalid_stream : public pdal_error
 
 void LasReader::processOptions(const Options& options)
 {
+    m_skipWktVLR = options.getValueOrDefault<bool>("skip_vlr_wkt", false);
```

Comment:
Don't think this is necessary, is it?  The code I'm going to commit for the other fix will take care of this.  Is this just a temp thing?